$NetBSD: patch-scripts_zm_in,v 1.5 2022/09/27 01:20:39 gdt Exp $

/etc/rc.d/init.d/functions doesn't commonly exist.

The LOCKFILE feature appears unused, and requires a /var/lock/subsys directory
we don't have.

--- scripts/zm.in.orig	2016-02-03 18:40:30.000000000 +0000
+++ scripts/zm.in
@@ -7,12 +7,12 @@
 # For systemd environments, use the ZoneMinder systemd unit file instead
 
 # Source function library.
-. /etc/rc.d/init.d/functions
+#. /etc/rc.d/init.d/functions
 
 prog=ZoneMinder
 ZM_CONFIG="@ZM_CONFIG@"
 pidfile="@ZM_RUNDIR@"
-LOCKFILE=/var/lock/subsys/zm
+#LOCKFILE=/var/lock/subsys/zm
 
 loadconf()
 {
@@ -54,7 +54,7 @@ start()
     RETVAL=$?
     [ $RETVAL = 0 ] && success || failure
     echo
-    [ $RETVAL = 0 ] && touch $LOCKFILE
+    #[ $RETVAL = 0 ] && touch $LOCKFILE
     return $RETVAL
 }
 
@@ -66,7 +66,7 @@ stop()
     RETVAL=$?
     [ $RETVAL = 0 ] && success || failure
     echo
-    [ $RETVAL = 0 ] && rm -f $LOCKFILE
+    #[ $RETVAL = 0 ] && rm -f $LOCKFILE
 }
 
 zmstatus()
@@ -107,7 +107,7 @@ case "$1" in
         result=`$ZM_PATH_BIN/zmdc.pl check`
         if [ "$result" = "running" ]; then
             $ZM_PATH_BIN/zmdc.pl shutdown > /dev/null
-            rm -f $LOCKFILE
+            #rm -f $LOCKFILE
             start
         fi
         ;;
